CSharpBasics/FinalTest/Tests/TrainingMainPage.cs

```diff
@@ -0,0 +1,86 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FinalTest.Tests
+{
+    public class TrainingMainPage : BaseTest
+    {       
+
+        private const string _mainLinkTrainingPage = "https://training.by/";
+
+        private const string _languageToggleXPath = "//*[contains(@class, 'menu-control language-control dropdown')";
+        private const string _languageDropDownXPath = "//*[contains(@class, 'language-control__menu dropdown-menu')]";
+        private const string _languagesInDropDownXPath = "//*[contains(@class, 'language-control__item')]";
+
+        private const string _enLanguageIconXPath = "//*[contains(@class, 'menu-control__toggle')] [contains(text(), 'EN')]";
+        private const string _ruLanguageIconXPath = "//*[contains(@class, 'menu-control__toggle')] [contains(text(), 'RU')]";
+        private const string _uaLanguageIconXPath = "//*[contains(@class, 'menu-control__toggle')] [contains(text(), 'UA')]";
+
+        private const string _enInDropDownXPath = "//a[contains(@class, 'language-control__item')][contains(text(), 'English')]";
+        private const string _ruInDropDownXPath = "//a[contains(@class, 'language-control__item')][contains(text(), 'Русский')]";
+        private const string _uaInDropDownXPath = "//a[contains(@class, 'language-control__item')][contains(text(), 'Українська')]";
+
+        public void TrainingMainPageIsOpened()
+        {
+            Driver.Url.Equals(_mainLinkTrainingPage);
+        }
+
+        public void OpenLanguagesDropDown()
+        {
+            Driver.FindElement(By.XPath(_languageToggleXPath)).Click();
+        }
+
+        public bool LanguagesDropDownIsOpened()
+        {
+            var dropDown = Driver.FindElement(By.XPath(_languageDropDownXPath));
+            return dropDown.Enabled;
+        }
+
+        public void SelectLanguageInDropDown(Languages languages)
+        {
+      
+            switch (languages)
```

Comment:
This approach can be used. But if you have 15 languages this approach will not be suitable. It is better to create a list of elements with languages ​​and a method in which we will pass the language as a variable and using LINQ return the element from the list. The same approach for IsPageWithSelectedLanguageIsOpened.